esmvalcore/preprocessor/_time.py

```diff
@@ -711,3 +754,73 @@ def timeseries_filter(cube, window, span,
                                weights=wgts)
 
     return cube
+
+
+def resample_hours(cube, hours):
+    """
+    Converts x-hourly data to y-hourly by eliminating extra timesteps.
+
+    Converts x-hourly data to y-hourly (y > x) by eliminating the extra
+    timesteps. This is intended to be used only with instantaneous values.
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        Input cube.
+    hours: int
+        The frequency of the desired data.
+
+    Returns
+    -------
+    iris.cube.Cube
+        Cube with the new frequency.
+
+    Raises
+    ------
+    ValueError:
+        The specified freuqncy is not a divisor of 24.
+    """
+    if 24 % hours != 0:
+        raise ValueError('The number of hours must be a divisor of 24')
```

Comment:
what happens if `hours = 24`?

Reply:
also can you pls add a test that tests for the raised exception :beer:

Reply:
Or what happens if `hours=0.5` or `hours=0.25`?